src/main/java/com/flipkart/ranger/healthcheck/HealthChecker.java

```diff
@@ -41,7 +45,7 @@ public void run() {
             try {
                 healthcheckStatus = healthcheck.check();
             } catch (Throwable t) {
-                logger.error("Error running healthcheck. Setting node to unhealthy");
+                logger.error("Error running healthcheck. Setting node to unhealthy", t);
```

Comment:
If it's anything other than for resources, it can be coding errors, retires will not help in that case. This function is anyways called after about a second. So doing retires in this context would rarely help.